src/test/java/com/philips/research/spdxbuilder/core/domain/ConversionInteractorTest.java

```diff
@@ -124,37 +127,50 @@ void setsDocumentIdentification() {
     }
 
     @Nested
-    class spdxDocument{
+    class spdxDocument {
         ByteArrayOutputStream spdxOutputStream = new ByteArrayOutputStream();
         Stream<String> lines;
+        DateTimeFormatter isoFormat = DateTimeFormatter.ofPattern("yyyy-MM-dd'T'HH:mm:ss'Z'")
+                .withZone(ZoneId.of("UTC"));
+        String dateString = "2000-01-01T01:01:01.111Z";
+        DateTimeFormatter parser = DateTimeFormatter.ofPattern("yyyy-MM-dd'T'HH:mm:ss.SSS'Z'");
+        LocalDateTime localDateTime = LocalDateTime.parse(dateString, parser);
+
         @BeforeEach
         void setUp() {
+            bom.setCreatedAt(localDateTime);
             bom.addPackage(project).addPackage(pkg);
             BomProcessor spdxWriter = new SpdxWriter(spdxOutputStream);
             interactor.apply(spdxWriter);
             lines = spdxOutputStream.toString(Charset.defaultCharset()).lines();
         }
 
-        @Test
-        void verifyPackageWithNoSupplierField () throws IOException {
-            Optional<String> packageSupplier = lines.filter(line -> line.startsWith("PackageSupplier: ")).findFirst();
-            assertThat(packageSupplier.get()).isEqualTo("PackageSupplier: NOASSERTION");
+        @AfterEach
+        void shutdown() throws IOException {
             spdxOutputStream.close();
         }
 
         @Test
-        void verifyPackageOrder () throws IOException {
+        void verifyPackageWithoutSupplierField() {
+            String packageSupplier = lines.filter(line -> line.startsWith("PackageSupplier: ")).findFirst().orElse("");
+            assertThat(packageSupplier).isEqualTo("PackageSupplier: NOASSERTION");
+        }
+
+        @Test
+        void verifyPackageOrder() {
             ArrayList<String> packages = new ArrayList<>(List.of("PackageName: Namespace/Project", "PackageName: Namespace/Package"));
             ArrayList<String> packageNames = lines.filter(line -> line.startsWith("PackageName: ")).collect(Collectors.toCollection(ArrayList::new));
             assertThat(packages).isEqualTo(packageNames);
-            spdxOutputStream.close();
         }
-    }
 
-
-
-//    @Nested
-//    class Curation {
+        @Test
+        void verifySBOMCreatedTime() {
+            String isoDate = isoFormat.format(localDateTime);
+            String created = lines.filter(line -> line.startsWith("Created: ")).collect(Collectors.joining(""));
+            assertThat(created).isEqualTo("Created: " + isoDate);
+        }
+    }
+        //    class Curation {
```

Comment:
This should not have an indentation and should stay alligned with remainder of the comment.